rotkehlchen/tests/unit/globaldb/test_migrations.py

```diff
@@ -11,17 +12,20 @@
 from rotkehlchen.globaldb.utils import GLOBAL_DB_VERSION
 from rotkehlchen.tests.utils.globaldb import patch_for_globaldb_migrations
 
+if TYPE_CHECKING:
+    from rotkehlchen.globaldb.handler import GlobalDBHandler
+
 
 @pytest.mark.parametrize('globaldb_upgrades', [[]])
 @pytest.mark.parametrize('run_globaldb_migrations', [False])
 @pytest.mark.parametrize('custom_globaldb', ['v4_global_before_migration1.db'])
-def test_migration1(globaldb):
+def test_migration1(globaldb: 'GlobalDBHandler'):
     """Test for the 1st globalDB data migration"""
     # Check state before migration
     with globaldb.conn.read_ctx() as cursor:
-        assert globaldb.get_setting_value('version', None) == GLOBAL_DB_VERSION
-        assert globaldb.get_setting_value('last_data_migration', None) is None
-        assert globaldb.get_setting_value('last_assets_json_version', None) == 72
+        assert globaldb.get_setting_value('version', -1) == GLOBAL_DB_VERSION
```

Comment:
why this change evereywhere here?

Reply:
I added the types and the default value is integer. SInce this is a test I won't modified the function to accept None and changed the default value to -1